src/DatePicker/DatePicker.web.js

```diff
@@ -1,52 +1,205 @@
+      return {
+        month,
+        year: month === 11 ? state.year - 1 : state.year,
+      };
     });
+  };
 
-    onConfirm(date);
+  swapMonthForward = () => {
+    this.setState(state => {
+      const month = (state.month + 1) % 12;
+      return {
+        month,
+        year: month === 0 ? state.year + 1 : state.year,
+      };
+    });
   };
 
   render() {
-    const { minDate, maxDate, customInput, mode } = this.props;
-    const { date } = this.state;
+    const { isVisible, minDate, maxDate, onDismiss } = this.props;
+    const { date, month, year } = this.state;
+
+    if (!isVisible) {
+      return null;
+    }
 
-    const showTimeSelect = mode === 'time' || mode === 'datetime';
-    const showTimeSelectOnly = mode === 'time';
+    const monthLength = new Date(year, month + 1, 0).getDate();
+    const startDay = new Date(year, month, 1).getDay();
+    const selectedDay = new Date(
+      date.getFullYear(),
+      date.getMonth(),
+      date.getDate()
+    );
+    const backDisabled =
+      minDate && year <= minDate.getFullYear() && month <= minDate.getMonth();
+    const forwardDisabled =
+      maxDate && year >= maxDate.getFullYear() && month >= maxDate.getMonth();
+
+    const dayTiles = [];
+    for (let i = 0; i < startDay + monthLength; i += 1) {
+      const dayId = i + 1 - startDay;
+      const tempDateTime =
+        dayId > 0 ? new Date(year, month, dayId).getTime() : 0;
+      const disabled =
+        tempDateTime < (minDate ?? 0) ||
+        tempDateTime > (maxDate ?? Number.MAX_SAFE_INTEGER);
+      dayTiles.push(
+        <DatePickerDayTile
+          key={`${year}-${month}-${i}`}
+          dayId={dayId}
+          selected={tempDateTime === selectedDay.getTime()}
+          disabled={disabled}
+          onDaySelect={this.handleDaySelect}
+        />
+      );
+    }
+
+    const weekRows = [];
+    while (dayTiles.length > 0) {
+      weekRows.push(
+        <View
+          key={`${year}-${month}-${weekRows.length}`}
+          style={styles.weekRow}
+        >
+          {dayTiles.splice(0, 7)}
+        </View>
+      );
+    }
 
     return (
-      <DatePicker
-        selected={date}
-        onChange={this.handleConfirm}
-        minDate={minDate}
-        maxDate={maxDate}
-        customInput={customInput}
-        showTimeSelect={showTimeSelect}
-        showTimeSelectOnly={showTimeSelectOnly}
-        timeIntervals={1}
-      />
+      <View style={styles.container}>
+        <TouchableWithoutFeedback onPress={onDismiss}>
+          <View style={styles.dismiss} />
+        </TouchableWithoutFeedback>
+        <View style={styles.picker}>
+          <View style={styles.head}>
+            <Button
+              width={40}
+              onPress={this.swapMonthBack}
+              disabled={backDisabled}
+              type="secondary"
+              label="<"
+            />
+            <View style={styles.headLabel}>
+              <Text>{MONTHS[month]}</Text>
```

Comment:
Definitely, thanks for noticing. I just added it to make this temporary version slightly nicer. 
But I can remove it or just replace it with month number for now.

Reply:
Month name removed and replace just with number for now.

Reply:
We will still need some way of localising these. I guess we could create some LocalisationProvider component inside the component library that would be needed if you want to localise the lib and it would default to English. 
We're talking future improvement here. No need to do it now :)